events.html

```diff
@@ -52,7 +52,11 @@
         <div class="col-sm-7 float-right-md-lg">
             <h1>Upcoming Events</h1>
             <div class="event-list">
-
+                <a class="event" href="https://www.eventbrite.ca/e/frostycon-csc-game-night-7-tickets-31724885988">
+                    <h3>IBM Hackathon & CSC Lan Party #7<b>iTel</b></h3>
+                    <h5>February 11th - 9:00am to February 12th - 2:00am</h5>
+                    Day Hackathon with IBM Watson/Bluemix from 9:00am to 5:00pm (lunch not provided). Boardgames and LAN setup 5:00 pm to 7:30pm, LAN Party 7:30pm to 2:00am, and Pizza at 8:30pm.
```

Comment:
Surely we can provide some better text here, explaining what the hackathon is about (more than just that it _has_ IBM Watson), and perhaps a little more on the LAN party. I wouldn't be explicit right here about lunch not being provided.